sass/directives/_carousels.scss

```diff
@@ -105,7 +105,7 @@
   font-size: $direction-arrow-size / 2;
   cursor: pointer;
   border: 1px solid $inactive-carousel-button-color;
-  padding: $base-padding / 2;
+  padding: #{$base-margin--small} #{($base-padding / 3)};
```

Comment:
ok I removed the padding, hardcoded the height/width and centered the arrows by using `text-align:center` and `line-height:28px`